nemesis/arenaPrep.py

```diff
@@ -490,23 +490,23 @@ def advanced_battle_logic():
                             # Double Gemini, two different locations
                             if memory.main.get_use_items_slot(42) < 100:
                                 battle.main.use_item(
-                                    memory.main.get_use_items_slot(42), rikku_flee=False
+                                    memory.main.get_use_items_slot(42), rikku_flee=True
```

Comment:
According to @coderwilson these all needed to be toggled, I have not actually tested this yet, but plan to either tonight or tomorrow.